src/zepben/auth/authenticator.py

```diff
@@ -30,6 +30,12 @@ class AuthMethod(Enum):
     An enum class that represents the different authentication methods that could be returned from the server's
     ewb/config/auth endpoint.
     """
+    @classmethod
+    def _missing_(cls, value):
```

Comment:
where does this get used?

Reply:
It overrides a method of `Enum` that gets called when it attempts to parse a string using the constructor but can't find the enum value. Solution is from stackexchange but it effectively makes the Enum constructor search for an uppercase version of the str in the enum if it can't find it by default.

Reply:
ok cool, didn't know that.